Dockerfile.mysql

```diff
@@ -0,0 +1,12 @@
+FROM mysql:latest
+
+ENV MYSQL_ROOT_PASSWORD=1234
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
```

Comment:
Using '1234' as the MYSQL_PASSWORD is a security risk. It's recommended to use a stronger, more complex password to protect your database.

Reply:
The password was set according to the task.